articles/overview/what-is-qsharp.md

```diff
@@ -30,7 +31,17 @@ Q# programs are often paired with a host program written in C# or Python, which
 
 Until now, to learn quantum computing you needed to learn the circuit model to understand the algorithms in the form of ordered sequences of quantum gates and measurements. With Q# you can take another path: learn quantum computing by writing quantum programs.
 
-Q# provides you with an increasing number of libraries and user-defined types that will help you to implement tools and functions already created by others to build advanced quantum algorithms. For example, you need to entangle two-qubits q1 and q2? Instead of applying individually the necessary gates to get the qubits entangled you can use the already built-in operation `PrepareEntangledState([q1], [q2])`.
+## Use Q# to design quantum algorithms
+
+Q# provides you with an increasing number of libraries and user-defined types that will help you to implement tools and build advanced quantum algorithms. For example, you need to entangle two-qubits q1 and q2? Instead of applying individually the necessary gates to get the qubits entangled you can use the already built-in operation `PrepareEntangledState([q1], [q2])`.
+
+## Use Q# to estimate quantum resources
+
+You can simulate the execution of your Q# program using the full state quantum simulator that provided with the Quantum Development Kit (QDK).  The QDK also provides resource estimators that give you insights on the performace of Q# programs that are too big to be run on a simulator.  This is highly valuable for algorithm designers, because they can tune their programs to use fewer resources (e.g. fewer number of qubits running for fewer numbers of operations), to be runnable when on earlier smaller scale quantum hardware.   
```

Comment:
```suggestion
You can simulate the execution of your Q# program using the full state quantum simulator that is provided with the Quantum Development Kit (QDK).  The QDK also provides resource estimators that give you insights on the performance of Q# programs that are too big to be run on a simulator.  This is highly valuable for algorithm designers, because they can tune their programs to use fewer resources (e.g. fewer number of qubits running for fewer numbers of operations), to run on earlier smaller scale quantum hardware.   
```